### PR TITLE
openai-base path updates

### PR DESCRIPTION
**Pull request by mabry1985**

### CODE REVIEW

**Comment by mabry1985** [comment]
A few regressions/bugs here from initial testing:
- Chat no longer has streaming replies
- Knowledge upload not working atm, no errors
- autocomplete seems to be okay for the most part, but I see the chat models often include the initial words. This is a problem I've definitly ran into before and why the instruct models seem better for these tasks imo

Screenshot 2023-10-07 at 1 12 22 PM

**Comment by mabry1985** [comment]
a couple regressions that I def want to figure out before merging your work, but overall it will be a solid addition to the codebase

**Comment by mabry1985** [comment]
@bioshazard I think the main thing I would like to sort out before merge is the embeddings. I'll take a pass at it this week

**Comment by bioshazard** [comment]
I did attempt it here and had a working kb ingest:

https://github.com/artificialcitizens/acai.so/pull/57/commits/e4ec23551b1dbf3450dade9bd99f60833769e2d3

Tho I think it might be nice to loop through the docs rather than bulk embed for local LLM. Locks it up till it finishes.

**Comment by bioshazard** [comment]
Might have regressed openai embed tho yea dang. Might be worth undoing the embed too get it in then trying for local embed again another pr if it gives too much trouble

**Comment by mabry1985** [comment]
@bioshazard good news! It seems like false alarms on the embeddings and the streaming chat 🎉  They are both working today apparently. Gonna run some more tests to make sure

File: src/lib/ac-langchain/models/chat.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  return {
+    chat: new ChatOpenAI({
+      modelName: 'gpt-3.5-turbo',
+      ...acaiDefaults,
+      ...fields,
+    }),
+  };
+}
+
+// I think we should consolidate to Chat in all cases, but this replaces non-chat for now
```
Comment (mabry1985) [inline src/lib/ac-langchain/models/chat.ts:33]: @bioshazard I think there are definitely cases where we want an instruct model. Especially with the future Huggingface integration, I see room for different instruction formats. Worth testing though

Comment (mabry1985) [reply src/lib/ac-langchain/models/chat.ts:33]: Happy with converting to chat at the moment for sure

**Pull request merged**